gomega_dsl.go

```diff
@@ -91,7 +91,7 @@ func InterceptGomegaFailures(f func()) []string {
 }
 
 //Ω wraps an actual value allowing assertions to be made on it:
-//	Ω("foo").Should(Equal("foo"))
+//    Ω("foo").Should(Equal("foo"))
```

Comment:
Switching from tab to space yields no visible change in output, but going for consistency

<img width="288" alt="screen shot 2018-06-12 at 5 47 25 pm" src="https://user-images.githubusercontent.com/1851748/41321460-a7b4dd72-6e69-11e8-9abb-493df83f7b96.png">
<img width="277" alt="screen shot 2018-06-12 at 5 47 46 pm" src="https://user-images.githubusercontent.com/1851748/41321461-a7c7abfa-6e69-11e8-9764-86d983b199b4.png">